### ISSUE TITLE
Cannot find format for "Query to limit resources"

### ISSUE THREAD

**Issue by davidcapatch** [Daniel-KM/Omeka-S-module-Timeline#15]
Hi, is there any documentation of formatting the "Query to limit resources"? I can't seem to figure out the fields that are usable or a method of formatting the string.
The only to fields I could manage were the default `site_id` or `id`  

I can use `id` to query for a single item, but cannot figure out how to query for multiple items by `id` or by any other field. Attempts included were
id=1,2  
id=1,id=2  
id=1&2  
id=1&id=2  

Any commas throw 500 errors and need to have the timeline deleted and made from scratch.

I do not have direct access to the database.
I also tried using one of the default Dublin Core fields that can be added to an item, e.g. `pmid` and using that to filter. But understanding that probably would need to result in a SQL `JOIN` to another table as it is not directly stored on the timeline item, but the extra metadata fields would probably be foreign keyed, one-to-many.

Any help is appreciated. Cheers!

![image (2)](https://user-images.githubusercontent.com/18700676/68913511-8682e400-0729-11ea-9316-43dd202b6bb6.png)

**Comment by Daniel-KM** [comment]
This is the query you get in an advanced search : see https://omeka.org/s/docs/user-manual/sites/site_pages/#browse-preview.

**Comment by Daniel-KM** [comment]
And it's not possible to set multiple ids, but you can use a property value in common.

**Comment by davidcapatch** [comment]
So this is what we determined. Based on your comment we stumbled upon the Item "Advanced Search". If you make a search of whatever parameters you want it gives you the url. You then copy and paste from the `?` and everything after into the "Query to limit resources" field on the timeline.

If I may make a suggestion, to place an example usage of this field and generating the url query string from the item advanced search page. It would be helpful to future users.

![image (3)](https://user-images.githubusercontent.com/18700676/68987763-03ca5980-07fb-11ea-923e-46dd9d42ed86.png)
